Mathlib/FieldTheory/Adjoin.lean

```diff
@@ -1244,26 +1244,72 @@ theorem Lifts.exists_upper_bound (c : Set (Lifts F E K)) (hc : IsChain (· ≤ 
 
 /-- Given an element `s : E` whose conjugates are all in `K`, any lift can be extended to one
   whose carrier contains `s`. -/
-theorem Lifts.exists_lift_of_splits (x : Lifts F E K) {s : E} (h1 : IsIntegral F s)
-    (h2 : (minpoly F s).Splits (algebraMap F K)) : ∃ y, x ≤ y ∧ s ∈ y.carrier :=
-  have I1 : IsIntegral x.carrier s := h1.tower_top
-  have I2 := (minpoly.degree_pos I1).ne'
-  have key : (minpoly x.carrier s).Splits x.emb.toRingHom :=
-    splits_of_splits_of_dvd _ (map_ne_zero (minpoly.ne_zero h1))
-      ((splits_map_iff _ _).2 (x.emb.comp_algebraMap ▸ h2)) (minpoly.dvd_map_of_isScalarTower _ _ _)
+theorem Lifts.exists_lift_of_splits' (x : Lifts F E K) {s : E} (h1 : IsIntegral x.carrier s)
```

Comment:
Perhaps the docstrings should be changed? The old `Lifts.exists_lift_of_splits` is "whose conjugates over `F` are all in `K`", and the new one is "whose conjugates over `x.carrier` are all in `K`".

Reply:
Thanks! This was an oversight. I shouldn't have modified the copy of the theorem that carries the docstring with it ...